browser-test/src/end_to_end_enumerators.test.ts

```diff
@@ -134,9 +134,13 @@ describe('End to end enumerator test', () => {
     await applicantQuestions.answerNameQuestion('Porky', 'Pig')
     await applicantQuestions.clickNext()
 
+    // Validate that enumerators are accessible
+    await applicantQuestions.validateAccessibility()
```

Comment:
I'd leave a comment here as to why we don't validate accessibility as a separate test like in the other files.

Reply:
Ah no good reason to not have it as a separate test :). Refactored this to match the others a bit more.